db/migrate/20200224095152_default_nil_fields_end_user_removal.rb

```diff
@@ -0,0 +1,13 @@
+class DefaultNilFieldsEndUserRemoval < ActiveRecord::Migration[5.0]
```

Comment:
You should have schemas files also committed to this migration

Reply:
Oops, right!